ceph/rados/core_workflows.py

```diff
@@ -2386,32 +2387,146 @@ def add_network_delay_on_host(
         log.info("Completed setting/unsetting the network delay and packet drops")
         return True
 
+    def configure_host_as_client(self, host_node):
+        """
+        Purpose of this module is to configure the ceph keyring and conf
+        on a cluster host which is not installer to run ceph commands from
+        it
+        Args:
+            host_node: node object of the host which needs to be converted
+        Returns: None
+        """
+        # copy /etc/ceph/ files to input host
+        ceph_conf, _ = self.node.shell(["cat /etc/ceph/ceph.conf"])
+        ceph_keyring, _ = self.node.shell(["cat /etc/ceph/ceph.keyring"])
+        host_node.exec_command(sudo=True, cmd="mkdir -p /etc/ceph")
 
-def configure_host_as_client(self, host_node):
-    """
-    Purpose of this module is to configure the ceph keyring and conf
-    on a cluster host which is not installer to run ceph commands from
-    it
-    Args:
-        host_node: node object of the host which needs to be converted
-    Returns: None
-    """
-    # copy /etc/ceph/ files to input host
-    ceph_conf, _ = self.node.shell(["cat /etc/ceph/ceph.conf"])
-    ceph_keyring, _ = self.node.shell(["cat /etc/ceph/ceph.keyring"])
-    host_node.exec_command(sudo=True, cmd="mkdir -p /etc/ceph")
-
-    for cont in [ceph_conf, ceph_keyring]:
-        file_name = (
-            "/etc/ceph/ceph.conf" if cont == ceph_conf else "/etc/ceph/ceph.keyring"
+        for cont in [ceph_conf, ceph_keyring]:
+            file_name = (
+                "/etc/ceph/ceph.conf" if cont == ceph_conf else "/etc/ceph/ceph.keyring"
+            )
+            file_ = host_node.remote_file(sudo=True, file_name=file_name, file_mode="w")
+            file_.write(cont)
+            file_.flush()
+            file_.close()
+
+        # Checking and installing ceph-common package on host
+        try:
+            out, rc = host_node.exec_command(
+                sudo=True, cmd="rpm -qa | grep ceph-common"
+            )
+        except Exception:
+            host_node.exec_command(sudo=True, cmd="yum install -y ceph-common")
+
+    def change_daemon_systemctl_state(
+        self, action, daemon_type: str, daemon_id: str, timeout=6000
+    ):
+        """
+        Retrieve logs for the requested daemon using journalctl command
```

Comment:
Please update the doc string to something about stop/start/restart of any daemons

Reply:
Updated the docstring